build/configs/rtl8730e/loadable_apps_xip/defconfig

```diff
@@ -8,6 +8,7 @@
 #
 # CONFIG_EXPERIMENTAL is not set
 # CONFIG_DEFAULT_SMALL is not set
+CONFIG_DOCKER_VERSION="2.0.0"
```

Comment:
Do we still need 2.0.0 with loadable XIP?

Reply:
It doesn't require newer toolchain. We can use older toolchain also.

Reply:
I recommend to use new toolchain, although there is no significant improvement from overall code size.
But along the way the toolchain version improve (ie. v6.3.1 -> v10.3.1), it includes some bug fix and optimization.